cypress/e2e/login.cy.tsx

```diff
@@ -48,4 +50,60 @@ describe('Button Component', () => {
     cy.get('.modal__btn').click();
     cy.get('.modal__btn').should('not.exist');
   });
```

Comment:
I recommend you add a blank space after each `it`, to make the whole file more legible.